src/bin/utilities/ringmesh-edit-infos.cpp

```diff
@@ -69,6 +70,32 @@ namespace
         geomodel_save( geomodel, geomodel_out_path );
     }
 
+    template < index_t DIMENSION >
+    void edit_interface_name( const std::string& geomodel_in_path,
+        const std::string& interface_old_name,
+        const std::string& interface_new_name,
+        const std::string& geomodel_out_path )
+    {
+        GeoModel< DIMENSION > geomodel;
+        geomodel_load( geomodel, geomodel_in_path );
+        GeoModelBuilder< DIMENSION > builder( geomodel );
+
+        for( const auto& ge_interface : geomodel.geol_entities(
+                 RINGMesh::GeologicalEntityType( "Interface" ) ) )
```

Comment:
RINGMesh:: is useless. I think it is preferable to use "Interface< DIMENSION >::type_name_static" instead of hard coded "Interface" if type name is changed in the future.